pkg/platform/yaml/yaml.go

```diff
@@ -158,6 +158,23 @@ func CreateYamlFile(ctx context.Context, Logger *zap.Logger, path string, fileNa
 	return false, nil
 }
 
+func FileExists(_ context.Context, Logger *zap.Logger, path string, fileName string) (bool, error) {
```

Comment:
Keep the Logger in small case -> logger and also put this function in utils of yaml.